ibis/expr/operations/relations.py

```diff
@@ -74,11 +75,7 @@ class SQLQueryResult(TableNode):
 class InMemoryTable(PhysicalTable):
     name = rlz.instance_of(str)
     schema = rlz.instance_of(sch.Schema)
-
-    @property
-    @abstractmethod
-    def data(self) -> util.ToFrame:
-        """Return the data of an in-memory table."""
+    data = rlz.instance_of(TableProxy)
```

Comment:
Would it make sense to move the base TableProxy implementation right here, before the InMemoryTable class definition?

Reply:
Yep, done.

Reply:
Thanks!